test/gtest/uct/ib/test_dc.cc

```diff
@@ -165,10 +165,14 @@ UCS_TEST_P(test_dc, dcs_single) {
     ucs_status_t status;
     uct_dc_mlx5_ep_t *ep;
     uct_dc_mlx5_iface_t *iface;
+    uint16_t first_allocated_dci;
 
     m_e1->connect_to_iface(0, *m_e2);
     ep = dc_ep(m_e1, 0);
     iface = dc_iface(m_e1);
+    first_allocated_dci = uct_dc_mlx5_iface_is_hybrid(dc_iface(m_e1)) ? 1 : 0;
```

Comment:
1. maybe use (UCT_DC_MLX5_HW_DCI_INDEX + 1) instead of 1?
2. first_allocated_dci -> first_sw_dci ?